lib/funky/url/parser.rb

```diff
@@ -0,0 +1,20 @@
+module Funky
+  # @api private
+  module URL
+    class Parser
+      attr_reader :url
+      VIDEO_ID_REGEXES = [/videos\/(\d*)/i, /v=(\d*)/i, /vb.\d*\/(\d*)/i]
```

Comment:
Should we check that `facebook.com` is present before this? I think http / https / www. are optional but we don't want to accept `youtube.com/videos/1234` as a valid Facebook video URL